spec/stack_spec.rb

```diff
@@ -96,6 +96,20 @@
     stack.delete
   end
 
+  it "gets status error reason" do
+    name = "spec-aws-ruby-stack-status-reason"
+
+    stack = new_template_one_stack(name: name)
+
+    begin
+      stack.create(params: {bucket_name: bucket_name, tag_value: "howdy**-"}, wait: true)
+    rescue 
+      expect(stack.latest_failed_events[0]).to include "The TagValue you have provided is invalid (Service: Amazon S3; Status Code: 400; Error Code: InvalidTag;"
```

Comment:
per our discussion, this needs to be `expect(stack.latest_failed_events[0]["status_reason"])`